server/src/main/java/io/druid/client/cache/MemcachedCache.java

```diff
@@ -440,9 +454,13 @@ public CacheStats getStats()
     );
   }
 
+  @Nullable
   @Override
   public byte[] get(NamedKey key)
   {
+    if (mode == Mode.WRITE_ONLY) {
+      return null;
```

Comment:
Would it break the processing if we throw exception instead of returning `null` when trying to get from `WRITE_ONLY`? I am thinking about the possible issue of silencing a mistake in configuration.

Reply:
Yes, it would break processing.

Reply:
Got it :+1: thanks